include/openssl/ssl.h.in

```diff
@@ -2366,6 +2366,47 @@ __owur int SSL_get_conn_close_info(SSL *ssl,
                                    SSL_CONN_CLOSE_INFO *info,
                                    size_t info_len);
 
+# define SSL_VALUE_CLASS_GENERIC                0
+# define SSL_VALUE_CLASS_FEATURE_REQUEST        1
+# define SSL_VALUE_CLASS_FEATURE_PEER_REQUEST   2
+# define SSL_VALUE_CLASS_FEATURE_NEGOTIATED     3
+
+# define SSL_VALUE_NONE                             0
+# define SSL_VALUE_QUIC_STREAM_BIDI_LOCAL_AVAIL     1
+# define SSL_VALUE_QUIC_STREAM_BIDI_REMOTE_AVAIL    2
+# define SSL_VALUE_QUIC_STREAM_UNI_LOCAL_AVAIL      3
+# define SSL_VALUE_QUIC_STREAM_UNI_REMOTE_AVAIL     4
+# define SSL_VALUE_QUIC_IDLE_TIMEOUT                0x8000
```

Comment:
Why does this one have such a different value than the rest?

Reply:
Mainly it's me trying to help compilers generate fast `switch()` code if we end up with a lot of options in the future. I'm probably overthinking it.

Reply:
I think you are overthinking it. I'd prefer to keep this as just sequential.